coding-concepts.js

```diff
@@ -20,24 +20,24 @@ const greeter = (name) => {
 }
 // console.log(greeter("LEARN Student"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: undefined
+// b) Verify and explain: The answer will be undefined because there is no return statement inside the function and we need a return statement in order to invoke the function. 
 
 // --------------------3) What will this log?
 
 const multipliedByTwo = [4, 5, 6, 7, 8].map((number) => number * 2)
 // console.log(multipliedByTwo)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: [8, 10, 12, 14, 16]
+// b) Verify and explain: My initial answer was correct because .map method is a higher order function built in method that is used on an array and returns an array with the same length. The .map method will iterate through the whole array and will take a value as a parameter which is the number in this case. Then, it will multiply the number by 2 through each iteration and will return an array with the original numbers multiplied by 2.
 
 // --------------------4) What will this log?
 
 const onlyOdds = [11, 12, 13, 14, 15].filter((number) => number % 2 !== 0)
 // console.log(onlyOdds)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: [11, 13, 15]
+// b) Verify and explain: My initial answer was correct because .filter method will iterate through the whole array and will return a new array based on the condition. The number will be the parameter and followed up with a conditional statement. The condition is that if the number is not evenly divisible by 2, then return an array with odd numbers.
```

Comment:
A little confusing at the end. You could say this instead: "...if the number is not evenly divisible by 2, keep the number in the returned array".